Hbase test (#48)
stackabletech/stackable-cluster: README.md

* [Maven](https://maven.apache.org/install.html)
* [(KubeAssert)](https://morningspace.github.io/kubeassert/docs/#/getting-started) currently saved in the project. Could be a krew plugin
  
# Where do I start?

1. Installing all the above tools
2. Start your local environment with ```./setup_local_environment.sh```
3. Execute the ```./run-tests.sh``` for the actual tests

# How to debug CI Pipeline?

> The CI Pipeline depends on T2 and runs on Jenkins.

Reviewer: I like the documentation of the process!

Author: Thank you!

For debugging purposes we use the directory ```.interactive```
1. Set your T2_Token created by Nikolaus ```export T2_TOKEN=<YOUR_TOKEN>```
2. Go to ```.interactive``` and run ```.startup.sh```
3. Open a new terminal session and list your running docker containers with ```docker ps```
4. Log into you docker container with ```docker exec -it <YOUR_CONTAINER_NAME> bash```
5. In case you lose your connection you can restore it: 
   1. check port in ```cat /root/.kube/config```
   2. Re-instate your ssh-tunnel ```stackable -i .cluster/key api-tunnel 6443```
   3. Check with ```kubectl get nodes``` to see if your tunnel works again

> Now you are in a container which is next to the created k8 cluster defined in ```.interactive/cluster.yaml```

Reviewer: ```suggestion
Now you are in a container which has access to the created k8 cluster defined in ```.interactive/cluster.yaml```
```
